package.json

```diff
@@ -22,16 +22,17 @@
   "author": "Tomas Della Vedova - @delvedor (http://delved.org)",
   "license": "MIT",
   "devDependencies": {
-    "fastify": "^0.40.0",
-    "sinon": "^4.2.0",
+    "fastify": "^2.10.0",
+    "node-fetch": "^2.6.0",
+    "sinon": "^7.5.0",
     "standard": "^10.0.3",
-    "tap": "^11.0.1",
-    "zipkin-context-cls": "^0.11.0",
-    "zipkin-transport-http": "^0.11.2"
+    "tap": "^14.10.2",
+    "zipkin": "^0.19.1",
+    "zipkin-context-cls": "^0.19.0",
+    "zipkin-transport-http": "^0.19.1"
```

Comment:
I think the zipkin dependencies should be dependencies, not devDependencies.

Reply:
Done! 👍